Completed/ModelExplorer/Assets/Scripts/Interactible.cs

```diff
@@ -14,7 +14,7 @@ public class Interactible : MonoBehaviour
 
     void Start()
     {
-        defaultMaterials = GetComponent<Renderer>().materials;
+        defaultMaterials = GetComponent<Renderer>()?.materials;
```

Comment:
I though Unity used C# 4, which doesn't have the null propagating operator (introduced in C# 6). When I try to use it, I get compiler errors. This compiles for you?